### DISCUSSION TITLE
New Jan 2023 project template documentation / help ?

### DISCUSSION THREAD

**Discussion by derMaaster** [unoplatform/uno#11266] [Q&A] [answered]
[This](https://platform.uno/blog/uno-platform-4-7-new-project-template-performance-improvements-and-more/) blogpost discusses the new project template - it indicates that MyApp is now the new "Shared Project" project. However there is also a folder created MyApp.Base.

MyApp has MainPage.xaml file, while MyApp.Base has AppHead.xaml file. Is Base a head? Where do I put my applications' business logic, and where does my UI code go? (xaml / WinUI/ UWP  type)

**Comment by jeromelaban** [comment, marked as answer]
See [this documentation page](https://platform.uno/docs/articles/uno-app-solution-structure.html) on the structure of apps.
